bin/cmds/enrich.js

```diff
@@ -1,93 +1,209 @@
+/* eslint-disable no-await-in-loop */
 /* eslint-disable camelcase */
 const fs = require('fs-extra');
 const path = require('path');
-const logger = require('../../lib/logger');
+const uuid = require('uuid');
 
-const { enrichmentFileCSV } = require('../enrich/csv');
-const { enrichmentFileJSON } = require('../enrich/json');
+const { connection, getConfig } = require('../../lib/axios');
 
-const getExtensionOfFile = (file) => {
-  const basename = file.split(/[\\/]/).pop();
+/**
+ * get the extention of a filename
+ * @param {String} filename
+ * @returns {String} extension
+ */
+const getExtensionOfFile = (filename) => {
+  const basename = filename.split(/[\\/]/).pop();
   const pos = basename.lastIndexOf('.');
   if (basename === '' || pos < 1) {
     return '';
   }
   return basename.slice(pos + 1);
 };
 
+/**
+ * start a csv file enrichment
+ * @param {Object} args commander arguments
+ * @param -f --file <file> - file which must be enriched
+ * @param -a --attributes <attributes> - attributes which must be enriched
+ * in graphql format. By default, all attributes are added
+ *
+ * @param -s --separator <separator> - separator of csv out file
+ * @param -o --out <out> - name of enriched file
+ * @param -v --verbose - logs how much lines are enriched
+ * @param -u --use <use> - use a custom config
+ */
+const enrichCSV = async (args) => {
+  const axios = await connection(args.use);
+  const config = await getConfig(args.use);
+  let out;
+  let separator;
+  if (!args.file) {
+    console.error('file expected');
+    process.exit(1);
+  }
+  const file = path.resolve(args.file);
+  const fileExist = await fs.pathExists(file);
+  if (!fileExist) {
+    console.error('file not found');
+    process.exit(1);
+  }
+  const typeOfFile = getExtensionOfFile(file);
+  if (typeOfFile !== 'csv') {
+    console.error(`${typeOfFile} is not suported for enrichCSV. Required .csv`);
+    process.exit(1);
+  }
+
+  if (args.separator) {
+    separator = args.separator;
+  } else {
+    separator = ',';
+  }
+
+  if (args.out) {
+    out = args.out;
+  } else {
+    out = 'out.csv';
+  }
+
+  if (!args.attributes) {
+    args.attributes = '';
+  }
```

Comment:
This can be shortened with the `||` operator

```suggestion
  separator = args.separator || ',';
  out = args.out || 'out.csv';
  args.attributes = args.attributes || '';
```